RadarSDK/RadarAPIClient.m

```diff
@@ -932,6 +932,50 @@ - (void)getMatrixFromOrigins:(NSArray<CLLocation *> *)origins
                     }];
 }
 
+- (void)sendEvent:(NSString *)name
+     withLocation:(CLLocation *_Nullable)location
+         metadata:(NSDictionary *_Nullable)metadata
+completionHandler:(RadarSendEventsAPICompletionHandler _Nonnull)completionHandler {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if (!publishableKey) {
+        return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
+    }
+
+    NSMutableDictionary *params = [NSMutableDictionary new];
+    params[@"id"] = [RadarSettings _id];
+    params[@"installId"] = [RadarSettings installId];
+    params[@"userId"] = [RadarSettings userId];
+    params[@"deviceId"] = [RadarUtils deviceId];
+    params[@"name"] = name;
+    params[@"metadata"] = metadata;
```

Comment:
there are def params that I'm missing here, things like `foreground` / `stopped` / `replayed`